lib/expressions/DateLiterals.ts

```diff
@@ -0,0 +1,65 @@
+import { TypeURL } from '../util/Consts';
```

Comment:
DateTimeLiterals would a name more consistent with the other naming.
Additionally, I don't see why they go into a separate file. If put separately, I suggest to put all literals separately instead, either in a single `Literals.ts` or with one file for each type. Of course, if Literals are separate, so should all the other Term subtypes to keep things consistent and simple.

Reply:
I think it might be worth it to make a `Literals.ts` file. I moved this to their own file because term was about to go over 350 lines.

Reply:
Hmm, it doesn't seem like that would help much so I'll just keep the big file :)